Tests/MongoSwiftSyncTests/SpecTestRunner/SpecTest.swift

```diff
@@ -189,7 +189,7 @@ extension SpecTestFile {
         // if connected to a replica set, use the provided client to execute killAllSessions on the primary.
         // if connected to a sharded cluster, use the per-mongos clients to execute killAllSessions on each mongos.
         switch try client.topologyType() {
-        case .single:
+        case .single, .loadBalancer:
```

Comment:
it's not clear to me what the right behavior would be to terminate all open transactions when testing against a load balancer, and isn't mentioned in the unified test runner spec. so for now I'm assuming this is a no-op, but maybe when implementing LB support we'll need to do something else here